### ISSUE TITLE
Make the "Test" function dialog a separate exe, and/or be able to open it from xSchedule

### ISSUE THREAD

**Issue by DoctorWiz** [xLightsSequencer/xLights#5042]
**Is your feature request related to a problem? Please describe.**
On my 'player' machine that runs xSchedule, I often need to pull up the "Test" function (from the Sequencer Tools menu) to check that something is working and is assigned to the correct controller, port, or DMX address.  Currently, I need to open the Sequencer only to get to the test dialog.  But xSchedule is already running, so if I could just get to it from there...

**Describe the solution you'd like**
To be able to run the Test feature independently, without having to open (and wait for) the Scheduler, or to be able to access it from xSchedule.

**Comment by cybercop23** [comment]
You can always use FPP.

**Comment by DoctorWiz** [comment]
No, want to use the most excellent xSchedule on one of the many old klunker Windoze machines I have without having to convert it to linux (NOT bashing linux) or shell out for a Raspberry Pi.  Not to mention using that same (player) machine to make last minute tweaks to the layout and/or sequences.
Not bashing the most excellent FPP either, but xSchedule is far more convenient for my setup.  And my shop/office is 200ft away thru 3 gates.
